virttest/utils_net.py

```diff
@@ -1486,6 +1486,38 @@ def set_guest_ip_addr(session, mac, ip_addr,
         raise IPAddrSetError(mac, ip_addr, err)
 
 
+def get_guest_default_gateway(session):
+    """
+    Get guest gateway from serial session for linux guest
+    :param session: serial session
+    :return: return the default gateway
+    """
+    cmd = "ip route | awk '/default/ { print $3 }'"
+    try:
+        output = session.cmd_output(cmd).strip()
+        logging.debug("Guest default gateway is %s" % output)
+    except process.CmdError:
```

Comment:
we should be caring for `aexpect.ShellError` and `aexpect.ShellTimeoutError` but not `process.CmdError`

Reply:
Thank you for your suggestion, update to:
    output = None
    try:
        output = session.cmd_output(cmd).strip()
        logging.debug("Guest default gateway is %s" % output)
    except aexpect.ShellError or aexpect.ShellTimeoutError:
        logging.error("Failed to get the guest's default Gateway")
    return output

Reply:
Thank you for your suggestion, updated as below:  
  output = None
    try:
        output = session.cmd_output(cmd).strip()
        logging.debug("Guest default gateway is %s" % output)
    except (aexpect.ShellError, aexpect.ShellTimeoutError):
        logging.error("Failed to get the guest's default Gateway")
    return output